pkg/prompts/prompts.go

```diff
@@ -109,6 +117,26 @@ func CapturePChainAddress(promptStr string) (string, error) {
 		return "", err
 	}
 
+	_, hrp, _, err := address.Parse(addressStr)
+	if err != nil {
+		return "", err
+	}
+	switch network {
+	case models.Fuji:
```

Comment:
We should push these checks into the validate function rather than capture. All of our prompt functions return err, but they never should, even when the user has bad input. (I actually don't know what can trigger that error)

Reply:
It might be a little ugly, you might need to make multiple validation functions and pick between them, but I think that's a better approach.

Reply:
Fixed by providing a `getValidationFunc` based on the `network` param